ckanext/aircan/tests/test_views.py

```
@@ -0,0 +1,16 @@
+
+import pytest
+
+import ckanext.aircan.validators as validators
+
+
+import ckan.plugins.toolkit as tk
+
+
+@pytest.mark.ckan_config("ckan.plugins", "aircan")
+@pytest.mark.usefixtures("with_plugins")
+def test_aircan_blueprint(app, reset_db):
```

Ruff (0.14.10)

Unused function argument: `reset_db`

(ARG001)